Index: controllersIMPL/IdentifyByMatriculaIMPL.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package controllersIMPL;\r\n\r\nimport model.Cliente;\r\nimport repositories.ClienteRpository;\r\nimport controllers.IdentifyByMatriculaController;\r\n\r\npublic class IdentifyByMatriculaIMPL implements IdentifyByMatriculaController {\r\n    private ClienteRpository clienteRpository;\r\n\r\n    @Override\r\n    public Cliente identifyByMatricula(String matricula) {\r\n        return clienteRpository.identificarPorMatricula(matricula);\r\n    }\r\n}\r\n
===================================================================
diff --git a/controllersIMPL/IdentifyByMatriculaIMPL.java b/controllersIMPL/IdentifyByMatriculaIMPL.java
--- a/controllersIMPL/IdentifyByMatriculaIMPL.java	
+++ b/controllersIMPL/IdentifyByMatriculaIMPL.java	
@@ -1,8 +1,8 @@
 package controllersIMPL;
 
+import controllers.IdentifyByMatriculaController;
 import model.Cliente;
 import repositories.ClienteRpository;
-import controllers.IdentifyByMatriculaController;
 
 public class IdentifyByMatriculaIMPL implements IdentifyByMatriculaController {
     private ClienteRpository clienteRpository;
Index: controllers/IdentifyByMatriculaController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package controllers;\r\n\r\nimport model.Cliente;\r\n\r\npublic interface IdentifyByMatriculaController {\r\n    public Cliente identifyByMatricula(String matricula);\r\n\r\n    }
===================================================================
diff --git a/controllers/IdentifyByMatriculaController.java b/controllers/IdentifyByMatriculaController.java
--- a/controllers/IdentifyByMatriculaController.java	
+++ b/controllers/IdentifyByMatriculaController.java	
@@ -4,5 +4,4 @@
 
 public interface IdentifyByMatriculaController {
     public Cliente identifyByMatricula(String matricula);
-
-    }
\ No newline at end of file
+}
